addons/hr_timesheet/report/project_report.py

```diff
@@ -18,7 +18,7 @@ class ReportProjectTaskUser(models.Model):
 
     def _select(self):
         return super()._select() +  """,
-                CASE WHEN COALESCE(t.allocated_hours, 0) = 0 THEN NULL ELSE LEAST((t.effective_hours * 100) / t.allocated_hours, 100) END as progress,
+                CASE WHEN COALESCE(t.allocated_hours, 0) = 0 THEN NULL ELSE LEAST(t.effective_hours / t.allocated_hours, 1) END as progress,
```

Comment:
But do we use the `percentage` widget in that report?

Reply:
no it's not used
before our changes
![image](https://github.com/odoo/odoo/assets/64937092/eba77c9b-ada6-4bf8-b5dc-48d5252a7f41)
after 100 -> 1.0

Reply:
Then it means we have to add the widget on the field to be able to use it, no? 🤷‍♂️